### DISCUSSION TITLE
Can you lock certain languages in localized fields?

### DISCUSSION THREAD

**Discussion by nehalist** [pimcore/pimcore#15377] [Q&A / Support]
Hey there,

I do import products via a custom command into my custom data objects. Products come with a default translation which should not be editable within Pimcore, but all other languages should be editable. Any idea how to achieve that?

Thanks!

**Comment by dvesh3** [comment]
You can manage language permissions based on workspaces by going to special settings
<img width="764" alt="image" src="https://github.com/pimcore/pimcore/assets/5137917/a00dd6b9-7d8d-4783-88e3-797919e77553">

and then selecting specific langauges in Edit (if none is selected then all are permitted)
<img width="309" alt="image" src="https://github.com/pimcore/pimcore/assets/5137917/49fa0783-9fb0-498d-a1d2-c8b483ea9a8a">
